packages/website/docs/examples/alert-block.md

```diff
@@ -0,0 +1,464 @@
+---
+title: Alert Block
+description: In this example, we create a custom Alert block which is used to emphasize text.
+imageTitle: Alert Block
+path: /docs/alert-block
+---
+
+<script setup>
+import { useData } from 'vitepress';
+import { getTheme, getStyles } from "../demoUtils";
+
+const { isDark } = useData();
+</script>
+
+# Alert Block
+
+In this example, we create a custom `Alert` block which is used to emphasize text.
+
+In addition, we create a Slash Menu item which inserts an `Alert` block and add an Alert item to the block type dropdown in the Formatting Toolbar.
+
+Finally, we add some theme-based styling to make our new block work nicely for both light and dark mode users.
+
+**Relevant Docs:**
+
+- [Custom Block Types](/docs/block-types#custom-block-types)
+- [Custom Slash Menu Item List](/docs/slash-menu#custom-slash-menu-item-list)
+- [Custom Formatting Toolbar](/docs/formatting-toolbar#custom-formatting-toolbar) & [Components](/docs/formatting-toolbar#components)
+- [Replacing UI Elements](/docs/ui-elements#replacing-ui-elements)
+- [Advanced: Overriding CSS](/docs/theming#advanced-overriding-css)
+
+::: sandbox {template=react-ts}
+
+```typescript-vue /App.tsx
```

Comment:
Nope, we need vue to access functions like `getTheme` and `getStyles` since we don't have access to `useData`, which is needed for those, in the sandbox directly